static/js/util/ProfileTab.js

```diff
@@ -16,14 +19,16 @@ class ProfileTab extends React.Component {
     // bind our field methods to this
     this.boundTextField = boundTextField.bind(this);
     this.boundSelectField = boundSelectField.bind(this);
+    this.boundStateSelectField = boundStateSelectField.bind(this);
     this.boundDateField = boundDateField.bind(this);
     this.editProfileObjectArray = editProfileObjectArray.bind(this);
 
     // options we set (for select components)
-    this.countryOptions = COUNTRIES.map(country => ({
-      value: country.Code,
-      label: country.Name
+    let countryOptions = Object.keys(iso3166.data).map(code => ({
+      value: code,
+      label: iso3166.data[code]['name']
```

Comment:
I generally like to put object literals with more than one item on multiple lines, but no reason other than style preference :)